tools/lan_choose_character_kart/lan_choose_character_kart.py

```diff
@@ -0,0 +1,247 @@
+from dataclasses import dataclass
+from tools.lan_choose_character_kart import symbols
+import subprocess
+import json
+import itertools
+from tools.gc_c_kit import devkit_tools
+import os
```

Comment:
The correct import order would be:

```suggestion
import itertools
import os
import subprocess
from dataclasses import dataclass

from tools.gc_c_kit import devkit_tools
from tools.lan_choose_character_kart import symbols
```

The convention is:

- Built-in modules
- Third-party modules
- Internal projects

Each group separated by a blank line and sorted asciibetically. And, within each group, `import` imports` before `from ... import ...` imports.

(Also noticed that `json` was not used.)